# Result of the executed action

string behavior_name
string status
uint8 result
uint8 answer

# Constants for action results
uint8 SUCCESS=0
uint8 FAIL=1
uint8 ERROR=2

# Constants for answers
uint8 NO_RESULT=0
uint8 NO_ANSWER=1
uint8 CORRECT=2
uint8 INCORRECT=3